onlineShop.core/Specials/InactiveSessionsDetector.cs

```diff
@@ -0,0 +1,35 @@
+﻿using onlineShop.Data.InMemory;
+using System;
+using System.Collections.Generic;
+using System.Timers;
+
+namespace onlineShop.core.Specials
+{
+    //public class InactiveSessionsDetector
```

Comment:
Add a TODO if you need to fix this commented code

Reply:
Added TODO